src/components/Pages/SetupPage/SetupPage.module.css

```diff
@@ -26,21 +26,21 @@
   align-items: center;
 }
 
-.left :global(.message) {
+:global(.message) {
   color: white !important;
   opacity: 70%;
 }
 
-.left :global(.ui.sub.header) > label {
+:global(.ui.sub.header) > label {
   color: white !important;
   opacity: 70%;
 }
 
-.left input {
+input {
```

Comment:
Does this work when using CSS modules?

Reply:
It does, I believe that css modules don't scope elements given that technically it would not be possible.

Reply:
<img width="341" alt="image" src="https://github.com/decentraland/auth/assets/24811313/e3f67c1b-16f6-4185-993e-e77cdfc8ab0b">

Reply:
I should still scope it with something to prevent it from overriding other inputs elsewhere